rpgJavaInterpreter-core/src/test/resources/mute/SIMPLE_MUTE_FAIL_EVALUATED_MESSAGE.rpgle

```diff
@@ -0,0 +1,8 @@
+     DA                S              8  0 INZ(5)
+     DB                S              8  0 INZ(8)
+     D RESULT          S              8  0 INZ(0)
+     D MSG             S             15    INZ('Failure message')
+      *
+    MU* FAIL[MSG]
+     C                   EVAL      RESULT = A + B
+     C                   SETON                                        LR
```

Comment:
I would add a newline at the end of the file

Reply:
OK